README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+<img src="https://github.com/jerrashi/hw01-1/blob/master/Profile%20Picture.jpg" width="200" height="200" />
```

Comment:
Nice HTML, but you can also use Markdown to embed images! Something like `![](https://github.com/jerrashi/hw01-1/blob/master/Profile%20Picture.jpg)` would've worked, or a relative path, if you had the picture on your laptop (and committed it).